Update cache-how-to-premium-persistence.md (#109363)
Azure/azure-content: articles/azure-cache-for-redis/cache-how-to-premium-persistence.md

    The first backup starts once the backup frequency interval elapses.
  
   > [!NOTE]

>    > When RDB files are backed up to storage, they are stored in the form of page blobs. If you are using storage account with HNS Enabled, persistence will tend to fail as page blobs aren't supported in Storage accounts with HNS enabled (ADLS Gen2) 

Reviewer: ```suggestion
   > When RDB files are backed up to storage, they are stored in the form of page blobs. If you're using a storage account with HNS enabled, persistence will tend to fail because page blobs aren't supported in storage accounts with HNS enabled (ADLS Gen2). 
```
